pkgs/applications/networking/mailreaders/sylpheed/default.nix

```diff
@@ -15,6 +15,17 @@ stdenv.mkDerivation rec {
     sha256 = "0j9y5vdzch251s264diw9clrn88dn20bqqkwfmis9l7m8vmwasqd";
   };
 
+  patches = [
+    (fetchurl {
```

Comment:
Please use fetchpatch and recalculate the hash.

Reply:
Thanks, updated the PR.